public/app/plugins/datasource/loki/querySplitting.ts

```diff
@@ -76,7 +76,7 @@ function adjustTargetsFromResponseState(targets: LokiQuery[], response: DataQuer
     .filter((target) => target.maxLines === undefined || target.maxLines > 0);
 }
 export function runSplitGroupedQueries(datasource: LokiDatasource, requests: LokiGroupedRequest[]) {
-  let mergedResponse: DataQueryResponse = { data: [], state: LoadingState.Streaming };
+  let mergedResponse: DataQueryResponse = { data: [], state: LoadingState.Streaming, key: uuidv4() };
```

Comment:
We already have this: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/querySplitting.ts#L222

Should we just reuse it here, it should be the same in all `requests`, shouldn't it?

Reply:
+1 absolutely.

Reply:
Done.